src/stdx.zig

```diff
@@ -876,7 +876,7 @@ test "Instant/Duration" {
     const instant_1 = Instant.now();
     const instant_2 = Instant.now();
     assert(instant_1.duration_since(instant_2).nanoseconds == 0);
-    assert(instant_2.duration_since(instant_1).nanoseconds > 0);
+    assert(instant_2.duration_since(instant_1).nanoseconds >= 0);
```

Comment:
Oh! That's so unfortunate :cry:

The problem with `>= 0` is that we can't really assert that we are measuring a duration or always zero!
I'd suggest a sleep gated by `if (builtin.os.tag == .windows)`.

Reply:
Or maybe, if  the goal is to assert `duration_since` we could just pass a dummy `instant_2` instead of `now()`.

Reply:
Good idea about using a dummy instance of instant, implemented!